bff-nav_ansatt/src/main/kotlin/no/nav/amt/tiltak/bff/nav_ansatt/NavAnsattControllerService.kt

```diff
@@ -50,7 +50,7 @@ class NavAnsattControllerService(
 		deltakerIder.addAll(alleEndringsmeldinger.map { it.deltakerId })
 		deltakerIder.addAll(alleVurderinger.map { it.deltakerId })
 
-		val deltakerMap = deltakerService.hentDeltakerMap(deltakerIder.distinct())
+		val deltakerMap = deltakerService.hentDeltakerMap(deltakerIder.distinct()).filterValues { !it.harAdressebeskyttelse() }
```

Comment:
Jeg synes det ville ha verdt å lage en eller to tester (gjerne integrasjonstester) på at de blir filtrert ut, bare sånn for sikkerhetsskyld.

Reply:
Ja, jeg kan sikkert det. Sitter veldig langt inne å lage tester i denne appen :( Men da vil jeg endre på koden også sånn at vi bare fjerner de som mangler i stedet for å kaste feil hvis endringsmeldinger gjelder deltakere som ikke finnes :)